<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMySliderWidClass</class>
 <widget class="QWidget" name="QtMySliderWidClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>45</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>218</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtMySliderWid</string>
  </property>
  <widget class="QLabel" name="label_max">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>30</y>
     <width>18</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QxtSpanSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_min">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>30</y>
     <width>18</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QxtSpanSlider</class>
   <extends>QSlider</extends>
   <header>qxtspanslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
